docs/source-app/glossary/storage/drive_content.rst

```diff
@@ -4,13 +4,24 @@
 About Drives
 ************
 
-Lightning Drive storage makes it easy to share files between LightningWorks so you can run your Lightning App both locally and in the cloud without changing the code.
+Lightning Drive storage makes it easy to share files between LightningWorks (Work) so you can run your Lightning App both locally and in the cloud without changing the code.
```

Comment:
```suggestion
Lightning Drives are shared app storage that allow you to share files between LightningWork components, so that you distributed components can share files when running on the cloud. Using drives, you can run your Lightning App both locally and in the cloud without changing the code.
```